packages/file-format/schema/layers/artboard.schema.yaml

```
@@ -13,6 +13,7 @@ allOf:
     properties:
       _class: { const: artboard }
       backgroundColor: { $ref: ../objects/color.schema.yaml }
+      didManuallyDisableResizingContent: { type: boolean }
```

I know we don't include `description` in many property schemas but since it's quite difficult to get you head around, maybe this is a good reason to provide more context. Also we can use this info later in any code generated from the schema.

```suggestion
      didManuallyDisableResizingContent: 
        type: boolean
        description: Some explanation how this relates to resizesContent
```

---

Perhaps like this?

```suggestion
      didManuallyDisableResizingContent:
        type: boolean
        description: If set to `true`, enabling a constraint on a child layer will not automatically enable `resizesContent`
```